drivers/display/display_stm32_ltdc.c

```diff
@@ -283,19 +339,37 @@ static int stm32_ltdc_init(const struct device *dev)
 	data->current_pixel_format = DISPLAY_INIT_PIXEL_FORMAT;
 	data->current_pixel_size = STM32_LTDC_INIT_PIXEL_SIZE;
 
+	k_sem_init(&data->sem, 0, 1);
+
+	config->irq_config_func(dev);
+
+#ifdef CONFIG_STM32_LTDC_DISABLE_FMC_BANK1
+	/* Clear MBKEN and MTYP[1:0] bits. */
+	FMC_Bank1_R->BTCR[0] &= ~(0x0000000D);
```

Comment:
Shouldn't this be `FMC_Bank1->BTCR[0] &= ~(0x0000000D);` ?
Otherwise I am getting a build error.

Reply:
`STM32H7x` and `STM32F7x` have different `FMC_Bank1_R_BASE` wrapper, add a soc check in here.